transport/matrix/transport.py

```diff
@@ -72,7 +70,7 @@
     make_room_alias,
     validate_and_parse_message,
     validate_userid_signature,
-    get_available_servers_from_config, get_server_url)
+    get_available_servers_from_config, get_server_url, UserAddressManager)
```

Comment:
please add 1 import per newline, and the closing `)` on another newline as well

Reply:
done